cmd/pilot-agent/main.go

```diff
@@ -220,6 +247,9 @@ func init() {
 		"IP Address and Port of a statsd UDP listener (e.g. 10.75.241.127:9125)")
 	proxyCmd.PersistentFlags().IntVar(&proxyAdminPort, "proxyAdminPort", int(values.ProxyAdminPort),
 		"Port on which Envoy should listen for administrative commands")
+	proxyCmd.PersistentFlags().IntVar(&infraAuthPolicy, "infraAuthPolicy", int(values.InfraAuthPolicy),
```

Comment:
Per discussion on 'upgradeability' and backward compat, I think it is important to start with both old port (8080) and new secure port on the server, and 'allowInsecure' set to true. 

This will allow us to test, compare performance, etc - and will follow the model of gradual adoption.